### ISSUE TITLE
Mini-Project 2 Feedback Issue - Opened by Instructor

### ISSUE THREAD

**Issue by michaelweylandt** [ayrama/STA9750-2024-FALL#5]
Thanks @ayrama !

Peer Feedback:

- @Timbila614
- @Chloe384
- @Cdiallo77
- @SethKauf

It is now time for the peer feedback round for Mini-Project 02. Please
review @ayrama's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini02.html#rubric,
please grade their submission out of a total of 50 points. Note that this
rubric is slightly different than that used for Mini-Project 01.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @ayrama, but you are of course
welcome to initiate a discussion.

@ayrama: please engage fully with your peers. They are here to help you!

Submission URL should be: https://ayrama.github.io/STA9750-2024-FALL/mp02.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by Chloe384** [comment]
Hi @ayrama,

Please see below for my feedback on your mini project 02.

**Written Communication: 9/10**

- Report was well written with headers that make your report flow naturally. Analysis was provided for each task. Some parts were kind of hard to follow due to the missing visuals (mainly task 3 & task 4).

**Project Skeleton: 10/10**

- All tasks were completed.

**Formatting & Display: 6/10**
 
- The tables for your output are difficult to read because the color of your text blends with the background color of your website. I suggest either changing your text color or the background color for the next projects. 
- I believe there are visual graphs and images (game poster) in your report looking at your code but I don't see it displayed. Maybe you didn't push those files to GitHub and therefore the visuals are missing. 

**Code Quality: 10/10**

- All codes look pretty accurate to me. 

**Data Preparation: 10/10**

**Total: 45/50**

**Comment by ayrama** [comment]
Thank you for pointing out that my graphs were missing from the repository (docs folder). I have now uploaded all the generated plots and JPG files. Additionally, I switched the theme from dark to a lighter one, which ensures that all DT tables are now clearly visible.
________________________________

**Comment by Chloe384** [comment]
Hi @ayrama,

Thank you for the update. I have reread your report and updated my comments for your mini project 02. I love your graphs and the game poster. Great job!

Best,
Chloe

**Comment by Cdiallo77** [comment]
Hello @ayrama, 

thank you for your submission

Written Communication: 10/10

Your report is very well written, key findings are highlighted and appropriately given context.

Project Skeleton: 9/10

Code completes all instructor-provided tasks correctly. Responses to open-ended tasks are particularly insightful and creative.

Formatting & Display: 10/10

I like the area graphs. I only saw you used them for this project and it is easy to read. Great job on this one.  

Code Quality: 10/10

Code quality is good, no issues there. 

Data Preparation: 10/10

Total: 49/50

Great Job!
